opal/corelib/range.rb

```diff
@@ -85,16 +93,26 @@ def exclude_end?
     @exclude
   end
 
-  alias :first :begin
+  def first(n=(n_specified=true; nil))
```

Comment:
Is that going to be `null` or `undefined` if we do `(n=undefined)`. Is `undefined` the same as ``` `undefined` ```?